From e9e8e17bd451bab595c8bd3c7c7368e6da764663 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 17 Sep 2017 17:56:15 +0200
Subject: [PATCH 1/1] efi_loader: capitalize EFI_LOCATE_SEARCH_TYPE values

Constants should be capitalized.
So rename the values of enum efi_locate_search_type.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/efi.h                 | 6 +++---
 lib/efi_loader/efi_boottime.c | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/include/efi.h b/include/efi.h
index 87b0b43f20..dc5e89c0b9 100644
--- a/include/efi.h
+++ b/include/efi.h
@@ -225,9 +225,9 @@ struct efi_time_cap {
 };
 
 enum efi_locate_search_type {
-	all_handles,
-	by_register_notify,
-	by_protocol
+	ALL_HANDLES,
+	BY_REGISTER_NOTIFY,
+	BY_PROTOCOL
 };
 
 struct efi_open_protocol_info_entry {
diff --git a/lib/efi_loader/efi_boottime.c b/lib/efi_loader/efi_boottime.c
index 0d234146d7..e52eec44c7 100644
--- a/lib/efi_loader/efi_boottime.c
+++ b/lib/efi_loader/efi_boottime.c
@@ -646,11 +646,11 @@ static int efi_search(enum efi_locate_search_type search_type,
 	int i;
 
 	switch (search_type) {
-	case all_handles:
+	case ALL_HANDLES:
 		return 0;
-	case by_register_notify:
+	case BY_REGISTER_NOTIFY:
 		return -1;
-	case by_protocol:
+	case BY_PROTOCOL:
 		for (i = 0; i < ARRAY_SIZE(efiobj->protocols); i++) {
 			const efi_guid_t *guid = efiobj->protocols[i].guid;
 			if (guid && !guidcmp(guid, protocol))
-- 
2.11.0

